modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/SubscriptionStore.java

```diff
@@ -814,6 +814,35 @@ public void deactivateClusterDurableSubscriptionsForNodeID(boolean isCoordinator
     }
 
     /**
+     * Marks all the durable subscriptions as inactive
+     *
+     * @throws AndesException Throw when updating the context store.
+     */
+    public void deactivateAllActiveSubscriptions() throws AndesException {
+
+        Map<String, String> subscriptions = andesContextStore.getAllDurableSubscriptionsByID();
+        Map<String, String> modifiedSubscriptions = new HashMap<>();
+
+        boolean subscriptionExists = false;
+
+        for (Map.Entry<String, String> entry : subscriptions.entrySet()) {
+
+            log.debug("Deactivating subscription with id: " + entry.getKey());
```

Comment:
IMO we should use a debug check[1] (if(log.isDebugEnabled()){..}) when doing this kind of getKey operation.
[1] http://stackoverflow.com/a/6504880/3844250